.github/workflows/ci-workflow.yml

```diff
@@ -177,58 +177,55 @@ jobs:
     container: pressiomodelreduction/${{ matrix.config.image }}
 
     env:
-      pressio_dir: /__w/pressio/pressio
-      pressio_builder_dir: /home/pressio_repos/pressio-builder
-      pressio_builds_dir: /home/pressio_builds
-      ctest_build: /home/pressio_builds/pressio/build
+      num_cpus: 4
+      pressio_src: /__w/pressio/pressio
+      pressio_build: /home/pressio_builds
+      eigen_inc_dir: /home/pressio_builds/eigen/install/include/eigen3
+      gtest_dir: /home/pressio_builds/gtest/install
+      trilinos_dir: /home/pressio_builds/trilinos/install
 
     steps:
     - uses: actions/checkout@v1
+
     - name: Creating environnement
       run: |
         apt-get update -y
         ${{ matrix.config.requirements }}
 
-    - name: Pressio Builder Update
+    - name: Configure
       run: |
-        cd ${{ env.pressio_builder_dir }}
-        git checkout main
-        git pull
+        cmake -S $pressio_src -B $pressio_build \
+          -D CMAKE_BUILD_TYPE:STRING=${{ matrix.build_type }} \
+          -D CMAKE_VERBOSE_MAKEFILE:BOOL=ON \
+          -D CMAKE_C_COMPILER:FILEPATH=/usr/bin/mpicc \
+          -D CMAKE_CXX_COMPILER:FILEPATH=/usr/bin/mpic++ \
+          -D MPI_Fortran_COMPILER:FILEPATH=/usr/bin/mpif90 \
+          -D PRESSIO_ENABLE_TESTS:BOOL=ON \
+          -D PRESSIO_ENABLE_TPL_EIGEN:BOOL=ON \
+          -D PRESSIO_ENABLE_TPL_MPI:BOOL=ON \
+          -D PRESSIO_ENABLE_TPL_TRILINOS:BOOL=ON \
+          -D PRESSIO_ENABLE_TPL_KOKKOS:BOOL=ON \
+          -D PRESSIO_ENABLE_TPL_BLAS:BOOL=ON \
+          -D PRESSIO_ENABLE_TPL_LAPACK:BOOL=ON \
+          -D PRESSIO_ENABLE_DEBUG_PRINT=ON \
+          -D EIGEN_INCLUDE_DIR=$eigen_inc_dir \
+          -D GTEST_ROOT=$gtest_dir \
+          -D TRILINOS_ROOT=$trilinos_dir \
+          -D KOKKOS_ROOT= \
+          -D KOKKOS_KERNELS_ROOT= \
```

Comment:
why do you need KOKKOS_ROOT and KOKKOS_KERNELS_ROOT?

Reply:
@antoinemeyer5

Reply:
I think we don't need those, sorry bad copy paste from https://github.com/Pressio/pressio/issues/617, I remove it